### ISSUE TITLE
Add support for timescaledb

### ISSUE THREAD

**Issue by akrauss** [RealOrangeOne/docker-db-auto-backup#75]
Hi,

please add support fpr timescaledb. Its a postgres-Database with timescale-addon.

https://hub.docker.com/r/timescale/timescaledb

The container can identified by
timescale/timescaledb
timescale/timescaledb-ha

Thanks! Great project!

**Comment by RealOrangeOne** [comment]
Am I right in seeing that it's just regular postgres with an extension? Does a regular `pg_dump` backup everything or is there extra data?

**Comment by akrauss** [comment]
you are right. a pg_dump database | psql database is all you need to backup and restore the database. the addon is included in the docker image.
